### PR TITLE
Updated index.md to change the grammatical mistakes

### PR DESCRIPTION
**Pull request by AkhilBabu94**
In the line no. 6, added an "an" before the word "element". Similar corrections in line no. 8, 9, and 10. In line no. 8, "then" is added in the line to make a proper meaning in completing the sentence. The word "to the" is added before the words such as "top", "left" and "bottom" in line nos. 9 and 10 to make a proper sentence.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

**Review by theNaut1lus — COMMENTED** [review]
@AkhilBabu94 , I have added some suggestions for grammatical changes, please consider reviewing so that we could merge this.

/cc: @RandellDawson

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@AkhilBabu94 I agree with @co16353sidak's assessment, so please make his recommended changes.

Thank you.

File: guide/english/css/properties/padding-property/index.md [hunk 1]
```diff
@@ -3,12 +3,12 @@ title: Padding Property
...
 
-* One value is provided it will apply to all sides of element
-* Two values are provided, first value will apply top and bottom sides and second value will apply to left and right sides of element.
-* Three values are provided, first value will apply to top, second value will apply left and right, third value will apply bottom sides of element.
-* Four values are provided, values will apply in the following order: top, right, bottom, left.
+* One value is provided then it will apply to all sides of an element.
+* Two values are provided, then first value will apply to the top and bottom sides and second value will apply to the left and right sides of an element.
+* Three values are provided, then first value will apply to the top, second value will apply to the left and right, third value will apply to the bottom sides of an element.
+* Four values are provided, then values will apply in the following order: top, right, bottom, left.
```
[the remark this answers is not in the document]
Comment (AkhilBabu94) [reply guide/english/css/properties/padding-property/index.md]: @co16353sidak i have added the required the required changes. Thanks

**Comment by RandellDawson** [comment]
@co16353sidak What do you think about @AkhilBabu94's new changes?

**Review by theNaut1lus — APPROVED** [review]
LGTM 🎉  - @RandellDawson 
Congratulations @AkhilBabu94 on your first contribution

**Review by RandellDawson — APPROVED** [review]

**Comment by AkhilBabu94** [comment]
Thanks all...😊

**Pull request merged**